### ISSUE TITLE
Friction and selectedAttraction behave differently on different refresh rates

### ISSUE THREAD

**Issue by timocouckuyt** [metafizzy/flickity#1175]
Hi guys,

First of all, you've done an amazing job with this library. Really outstanding, compared to others out there. We couldn't be more grateful!

One small "issue" we've noticed is that, with the same friction and selectedAttraction values (example below uses selectedAttraction: 0.022 and friction: 0.28), a carousel behaves differently viewing them on screens with different refresh rates.

2 video's to clearly illustrate this, below:

- Behaviour at 60Hz: https://devplus.be/downloads/flickity/60Hz.mov
- Behaviour at 144Hz: https://devplus.be/downloads/flickity/144Hz.mov

Since the animations are based on physics rather than time based transitions, it seems that on higher refresh rate screens, animations are much faster.

This may seem like a minor issue, but it's concerning for us because it creates different behaviours on the same browsers, same devices but with different screens. Where time based transitions are more controllable.

**Comment by desandro** [comment]
Yup, this is how Flickity was coded. It's based off framerate not actual time.

**Add a 👍  reaction** to this issue if you would like to see Flickity get time-based physics rather than frame-based. Do not add +1 comments — [They will be deleted](https://metafizzy.co/blog/use-github-reactions-delete-plus-1-comments/).

**Comment by Hlsgs** [comment]
Would this also cover #46 ?
